lib/kitchen/directions/eoc_section_title_link_snippet.rb

```diff
@@ -16,7 +16,7 @@ def self.v1(page:, title_tag: 'h3', wrapper: 'link')
         end
 
         title_snippet = <<~HTML
-          <#{title_tag} data-type="document-title" id="#{page.title.copied_id}">
+          <#{title_tag} data-type="document-title" id="#{page.copied_id}_autogentitlefrombaking">
```

Comment:
```suggestion
          <#{title_tag} data-type="document-title">
```

I'm going to try this and see if it works

Reply:
Deleting the ids off of every title link snippet is a huge change, heads up, and will need a pretty in-depth review before styles thinks about merging it